pkg/datapath/linux/routing/routing.go

```diff
@@ -58,7 +59,8 @@ func (info *RoutingInfo) Configure(ip net.IP, mtu int, masq bool) error {
 		Mask: net.CIDRMask(32, 32),
 	}
 
-	// Route all traffic to the ENI address via the main routing table
+	// Route all traffic to the ENI address via the main routing table.
+	// Ingress rules are an exception to the above comment.
```

Comment:
Ah, I recall now what happened. I initially had a different implementation than what this PR has and when I cherry-picked the commit here, I forgot to update this comment because it doesn't refer to anything correctly. You're correct that if I change it to say "egress" it would be accurate.